drools-yaml-rules-runtime/src/main/java/org/drools/yaml/runtime/domain/Source.java

```diff
@@ -1,4 +1,4 @@
-package org.drools.yaml.core.domain;
+package org.drools.yaml.runtime.domain;
 
 public class Source {
```

Comment:
This class belongs to the domain of the rule definition part, I don't understand why it should be in a different module.